bin/BuildPackages.sh

```diff
@@ -279,8 +248,8 @@ build_one_package() {
   ) || build_fail
 }
 
-date >> "$LOGDIR/$FAILPKGFILE.log"
-for PKG in "${PACKAGES[@]}"
+date >> "$LOGDIR/fail.log"
+for PKG in ${PACKAGES}
```

Comment:
When you say "packages with spaces", I assume you mean "package directories with spaces", and then I think you should drop the "package" and just ask for support for directories with spaces... I suggest opening an issue for that.